Source/Core/Core/HW/Wiimote.cpp

```diff
@@ -23,16 +27,17 @@
 // Limit the amount of wiimote connect requests, when a button is pressed in disconnected state
 static std::array<u8, MAX_BBMOTES> s_last_connect_request_counter;
 
-namespace WiimoteCommon
+namespace
 {
 static std::array<std::atomic<WiimoteSource>, MAX_BBMOTES> s_wiimote_sources;
+static std::optional<size_t> s_config_callback_id = std::nullopt;
 
-WiimoteSource GetSource(unsigned int index)
+static WiimoteSource GetSource(unsigned int index)
```

Comment:
This is already in an anonymous namespace so there's no need to use `static`. (that said, static does have the benefit of making it obvious which functions have internal linkage, but in that case the anonymous namespace is redundant)

Reply:
Removed.

Reply:
`OnSourceChanged` also doesn't need `static` (or did you mean to keep it?)

Reply:
Oh, I thought I removed it...